app/views/layouts/__header.html.erb

```diff
@@ -0,0 +1,48 @@
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
```

Comment:
Die Libs zu denen man Abhängigkeiten hat, bitte zentral über Rails-Pipeline und application-Files (.erb, .scss, .js) einbinden. 
Keinen direkten Aufruf von externen Libs, schon gar nicht pro einzelne Seiten.

https://edgeguides.rubyonrails.org/asset_pipeline.html

Reply:
ich verstehe dass das nicht in die Headerdatei soll, aber nicht die rails doku zur pipeline ... 
soll ich das in vendor/assets/javascript packen und dann irgendwie in den application header rendern?

Reply:
Du musst Bootstrap 4 oder besser 5 über Webpacker ausliefern und über Yarn installieren. Schaue dazu online Tutorials an. Sonst schauen wir es Montag noch mal an.